### PR TITLE
[MIG][11.0]hr_employee_phone_extension:Migration to 11.0.

### PR DESCRIPTION
**Pull request by Trivedi-Vacha-SerpentCS**
[MIG][11.0]hr_employee_phone_extension:Migration to 11.0.

### CODE REVIEW

File: hr_employee_phone_extension/README.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (JayVora-SerpentCS) [inline hr_employee_phone_extension/README.rst]: https

File: hr_employee_phone_extension/README.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Contributors
+------------
+
+* Sandy Carter <sandy.carter@savoirfairelinux.com>
+* El Hadji Dem <elhadji.dem@savoirfairelinux.com>
+* Vincent Renaville <vincent.renaville@camptocamp.com>
+* Charbel Jacquin <charbel.jacquin@camptocamp.com>
+* Denis Leemann <denis.leemann@camptocamp.com>
+
```
Comment (JayVora-SerpentCS) [inline hr_employee_phone_extension/README.rst]: Add SerpentCS

File: hr_employee_phone_extension/README.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+* update the module list
+* search for "Employee Phone Extension" in your addons
+* install the module
+
+Usage
+=====
+
+On the employee form view you will have 2 separate fields, one for Internal Number,
+second for Short Number, both optional.
+
```
Comment (JayVora-SerpentCS) [inline hr_employee_phone_extension/README.rst:28]: Add Runbot URL

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

File: hr_employee_phone_extension/README.rst (hunk 3 above)
Comment (Trivedi-Vacha-SerpentCS) [reply hr_employee_phone_extension/README.rst:28]: Done

File: hr_employee_phone_extension/README.rst (hunk 2 above)
Comment (Trivedi-Vacha-SerpentCS) [reply hr_employee_phone_extension/README.rst]: Done

**Review by JayVora-SerpentCS — APPROVED** [review]

**Review by novawish — APPROVED** [review]

**Pull request merged**